### PR TITLE
List Content Pack Installations

### PR DESCRIPTION
**Pull request by kmerz**
## Description
Add the list of content pack installations together with a button to show details and
uninstall content packs.
This details of the installation still miss vital parts like 'installed entitites' or 'set parameters',
since the server does not yet send this informations.

## How Has This Been Tested?
Installed a content pack and checked for the installations list.

## Screenshots (if appropriate):
screenshot_2018-07-24 graylog - content packs
screenshot_2018-07-24 graylog - content packs 1

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Row, Col } from 'react-bootstrap';
+
+import 'components/content-packs/ContentPackDetails.css';
+
+const ContentPackInstallView = (props) => {
+  const { comment } = props.install;
+  const createdAt = props.install.created_at;
```
Comment (dennisoelkers) [inline graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx]: You can shorten this by using destructuring:

```
const { comment, createdAt, createdBy } = props.install;
```

File: graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const createdBy = props.install.created_by;
+  return (<div>
+    <Row>
+      <Col smOffset={1}>
+        <div>
+          <dl className="deflist">
+            <dt>Comment:</dt> <dd>{comment}</dd>
+            <dt>Installed by:</dt> <dd>{createdBy}&nbsp;</dd>
+            <dt>Installed at:</dt> <dd>{createdAt}&nbsp;</dd>
```
Comment (bernd) [inline graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx]: Please use the `Timestamp` component to format the timestamp. This allows us to format the timestamp in the users/browser timezone.

See: https://graylog2.github.io/frontend-documentation/#timestamp

File: graylog2-web-interface/src/components/content-packs/ContentPackInstallastions.test.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```
Comment (bernd) [inline graylog2-web-interface/src/components/content-packs/ContentPackInstallastions.test.jsx]: This file has a typo in its name:

`ContentPackInstallastions.test.jsx`
`ContentPackInstallations.test.jsx`

File: graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx (hunk 1 above)
Comment (kmerz) [reply graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx]: Not exactly:

`const { comment, created_at, created_by } = props.install;`

But the would generate a bunch of warnings I would have to silent. Which I would not like to do.

**Pull request merged**